stubs/django/db/models/deletion.pyi

```diff
@@ -0,0 +1,25 @@
+from typing import Collection
+
+from django.db.models.base import Model
+from django.db.models.fields import Field
```

Comment:
The `Field` import errors are because the file defining those fields are [explicitly a lie](https://github.com/facebook/pyre-check/blob/5a85dd491427553b1799a2789ced4e76252e7fdd/stubs/django/db/models/fields/__init__.pyi#L6-L21). I think we can work around it by defining `Field` as a `Union` of all the field types. It's ugly, but I think it'll work.